### PR TITLE
attempted fix for mobile screens, removed sm:hidden param

### PR DESCRIPTION
**Pull request by Corvu-s**
Implements #18 

## Proposed Changes
  -Removed some css that made content hidden based on screen width

## Estimated Review Time
1-2 mins
 Here is the tailwind docs that i was referring to [https://tailwindcss.com/docs/responsive-design] pls try on your phones, it works for me.

### CODE REVIEW

File: project-webpage/pages/index.js [hunk 1]
```diff
@@ -18,9 +18,9 @@ export default function Home() {
       <nav className="bg-gray-800">
```
Comment (ReubenMathew) [inline project-webpage/pages/index.js:18]: Could you delete the code comment above? Other than that these fixes look good.

**Comment by ReubenMathew** [comment]
I'll check on my phone soon.

File: project-webpage/pages/index.js (hunk 1 above)
Comment (Corvu-s) [reply project-webpage/pages/index.js:18]: f just saw this

**Review by ReubenMathew — APPROVED** [review]
Nice!

**Pull request merged**